### ISSUE TITLE
A new participant

### ISSUE THREAD

**Issue by AlihanKayabas** [VoidmatrixHeathcliff/Civilization-Simulation#1]
Hey I would like to participate in this project. I'm a bioinformatics student in France.
This project is interesting to me: Simulating a civilization or even a small population is very interesting and rewarding.

Please contact me at malihankayabas@gmail.com

**Comment by VoidmatrixHeathcliff** [comment]
Hello, I'm sorry to reply so late. I haven't paid attention to this project since it was founded. I didn't find it until recently when I was searching for my past history. It's really a cool and wonderful project, and I'm still working hard for it.
However, the realization of a civilization can have multiple understandings at different levels. Writing a fantasy novel or making a realistic game can be regarded as a simulation of civilization. As for the specific implementation of this project, I'm still thinking about it. Welcome to join our immature organization(https://github.com/cardinaldevgroup), or focus on the engine(https://github.com/VoidmatrixHeathcliff/EtherEngine) developed by myself. This may be a good start.
Sorry for the late reply again, looking forward to your reply~

**Comment by AlihanKayabas** [comment]
Thanks for your reply. Is the project entirely written in C++? Or are there parts in Python?

**Comment by VoidmatrixHeathcliff** [comment]
No, the language only involves the implementation of the program. In my opinion, this project should be more about the design. The programming can support multiple languages (or any Turing complete language can do it). About the self-development engine recommended to you（ https://github.com/VoidmatrixHeathcliff/EtherEngine ）, although its source code is written in C + +, But the logic code based on it is written in Lua language, which is similar to Python language. If you master Python language, you will easily learn Lua scripting language.
Although the current version of the engine is cross platform, but due to the problem of developing platform, I only provide release version under Windows platform. If your computer is Windows system, please try it, or recompile it under your own platform (this may cause some trouble）, the QuickStart document may not match the latest release version, you can try a low version of the binary or wait for the quick start document update.
I'm very sorry that the complete version of the document is only in Chinese, and we are trying our best to translate it into other languages. Of course, you are also very welcome to participate in the process of engine document translation!
Sincerely invite you!
